guides/common/modules/proc_importing-a-custom-gpg-key.adoc

```diff
@@ -69,8 +69,9 @@ $ scp ~/etc/pki/rpm-gpg/RPM-GPG-KEY-_EXAMPLE-95_ root@{foreman-example-com}:~/.
 +
 [options="nowrap" subs="+quotes"]
 ----
-# hammer gpg create \
---key ~/RPM-GPG-KEY-_EXAMPLE-95_ \
---name "_My_Repository_" \
---organization "_My_Organization_"
+# hammer content-credentials create \
+--content-type gpg_key \
```

Comment:
Don't we usually indent the new line of a command to make it clear it's one command?

Reply:
We don't as far as I know.